draft-ietf-ppm-dap.md

```diff
@@ -2297,36 +2362,50 @@ MUST abort the aggregation job.
 
 Otherwise, the Leader proceeds as follows with each report:
 
-1. If the inbound prep response type is "continue" and `state` is
-   `Continued(prep_state)`, then the Leader computes
+1. If `state` has type `Continued` and the inbound `PrepareResp` has
+   type "continue", then the Leader computes
 
    ~~~ pseudocode
-   (state, outbound) = Vdaf.ping_pong_leader_continued(
+   state = Vdaf.ping_pong_leader_continued(
        "dap-15" || task_id,
        agg_param,
        state,
        inbound,
    )
    ~~~
 
-   where `task_id` is the task ID and `inbound` is the message payload. If
-   `outbound != None`, then the Leader stores `state` and `outbound` and
-   proceeds to another continuation step. If `outbound == None`, then the
-   preparation process is complete: either `state == Rejected()`, in which case
-   the Leader rejects the report and removes it from the candidate set or
-   `state == Finished(out_share)`, in which case the Leader commits to
-   `out_share` as described in {{batch-buckets}}. If commitment fails, then the
-   Leader rejects the report and removes it from the candidate set.
+   where `task_id` is the task ID and `inbound` is the payload of the inbound `PrepareResp`.
+   It then processes the next state transition:
+
+   * If the type of the new `state` is `Continued` or
+     `FinishedWithOutbound`, then the Leader stores `state` and proceeds
+     to the next continuation step.
 
-1. Else if the type is "finish" and `state == Finished(out_share)`, then
-   preparation is complete and the Leader stores the output share for use in
-   the collection interaction ({{collect-flow}}).
```

Comment:
@tgeoghegan we missed a "stores the output share" -> "commits the output share" here.

Reply:
Glad you spotted it